lib/preact/index.js

```diff
@@ -14,9 +14,9 @@ var alias = require('alias');
 
 var Preact = module.exports = integration('Preact')
   .assumesPageview()
-  .global('_lnq')
+  .global('_preactq')
```

Comment:
Just to make sure I understand—loading the Preact library now declares two globals, `_lnq` and `_preactq`, correct? If so, we should declare both globals by adding `.global('_lnq')` here.